pkg/controller/restore_controller.go

```diff
@@ -531,10 +531,13 @@ func (c *restoreController) runRestore(
 	}
 	defer closeAndRemoveFile(resultsFile, c.logger)
 
+	pluginManager := c.newPluginManager(log)
+	defer pluginManager.CleanupClients()
+
 	// Any return statement above this line means a total restore failure
 	// Some failures after this line *may* be a total restore failure
 	log.Info("starting restore")
-	restoreWarnings, restoreErrors = c.restorer.Restore(log, restore, info.backup, backupFile, actions)
+	restoreWarnings, restoreErrors = c.restorer.Restore(log, restore, info.backup, info.backupStore, backupFile, actions, pluginManager)
```

Comment:
I'm thinking that instead of passing an `info.backupStore`, the controller should just download the volume snapshot data and pass that, since it's already doing that for the backup tarball itself -- unless there's a reason not to that I'm missing